### PR TITLE
Create Vault table and db operations

### LINKED ISSUE #228: [Backend] Add new model, Deposit(wallet_id/amount/symbol)
🚨 **Attention: Please read the guidelines before applying; otherwise, you will be ignored.** 🚫📋
0. Please, join the telegram group to keep in touch https://t.me/+m0IGRiBvLtsyMzIy 
1. Read guideline [here](https://gist.github.com/djeck1432/f722f5fe518c80b248ff609627da25e5)
2. Create new model `Vault` for db in this [file](https://github.com/djeck1432/spotnet/blob/main/web_app/db/models.py) with next fields:
- user: ForeignKey(User)
- symbol: str
- amount: str
3. Add new class `DepositDBConnector` to `DBconnector` with methods 
 - Create new instance `create_vault`
 - Add balance for the user (by wallet_id) `get_vault_balance`
4. Run migrations( how to run migration read README.md file)

### PR DESCRIPTION
**Pull request by codebestia**
Fixes #228

### CODE REVIEW

File: web_app/db/crud.py [hunk 1]
```diff
@@ -618,25 +618,96 @@ def set_notification_allowed(
...
+        :param symbol: Token symbol or address
+        :param amount: An amount in string
+
+        :return: Vault
+        """
+        vault = Vault(user_id=user.id, symbol=symbol, amount=amount)
+        self.write_to_db(vault)
+        return vault
+
+    def get_or_create_vault(self, wallet_id: str, symbol: str) -> Vault | None:
```
Comment (djeck1432) [inline web_app/db/crud.py:666]: Please, rework this method into `get_vault` only
This method is overcomplicated

**Review by djeck1432 — CHANGES_REQUESTED** [review]
Looks pretty good, left one comment

File: web_app/db/crud.py (hunk 1 above)
Comment (codebestia) [reply web_app/db/crud.py]: Resolved.

**Review by djeck1432 — APPROVED** [review]
Great job, approved

**Pull request merged**